packages/@remirror/core/src/builtins/__tests__/persistent-selection-extension.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { extensionValidityTest, renderEditor } from 'jest-remirror';
+
+import { PersistentSelectionExtension } from '..';
+
+extensionValidityTest(PersistentSelectionExtension);
+
+describe('configuration', () => {
+  [
+    {},
+    { persistentSelectionClass: undefined },
+    { persistentSelectionClass: '' },
+    { persistentSelectionClass: 'selection' },
+  ].forEach((configuration) => {
+    test(`uses provided configuration (${JSON.stringify(configuration)}`, () => {
+      const {
+        add,
+        dom,
+        selectText,
+        nodes: { p, doc },
+      } = renderEditor([], { builtin: configuration });
+
+      add(doc(p('Some text')));
+      selectText({ from: 4, to: 9 });
+
+      expect(dom.outerHTML).toMatchSnapshot();
+    });
+  });
+});
+
+describe('focus', () => {
+  test('retains selection and decoration', () => {
+    const builtin = { persistentSelectionClass: 'selection' };
+    const editor = renderEditor([], { builtin });
+    const {
+      add,
+      dom,
+      selectText,
+      nodes: { p, doc },
+    } = editor;
+
+    add(doc(p('Some text')));
+    const range = { from: 4, to: 9 };
+    selectText(range);
+
+    // Verify that the selection has been applied
+    expect(editor.state.selection.from).toBe(range.from);
+    expect(editor.state.selection.to).toBe(range.to);
+
+    // Add a new element and focus that one
```

Comment:
You can also use `editor.view.dom.blur()`.

Reply:
I needed to explicitly cast the `dom` property to `HTMLElement` to have access to `blur()`, but this definitely reads better and makes it more obvious what the point of the test is.

Reply:
Oh interesting, try `editor.dom`, just realised that might already be cast to an HTML element.

Reply:
Ignore that, it isn't. I'll update it so that it is.